torch/ao/quantization/qconfig.py

```diff
@@ -223,13 +223,13 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
 Default qconfig for operators that reuse the observers from input Tensor, e.g. reshape
 """
 
-def get_default_qconfig(backend='fbgemm', version=0):
+def get_default_qconfig(backend='x86', version=0):
```

Comment:
can we split changing the default into a separate PR, so it can be easily reverted if this breaks something?

Reply:
OK. It's reverted now.